.github/workflows/build.yml

```diff
@@ -44,6 +44,41 @@ jobs:
 
     - run: yarn --frozen-lockfile
 
+  ui-test:
```

Comment:
Maybe add some small instructions or steps on how to run these new tests locally, in the root README?